imgproc_test.go

```diff
@@ -54,6 +54,11 @@ func TestConvexity(t *testing.T) {
 		t.Errorf("Invalid ContourArea test: %f", area)
 	}
 
+	convex := IsContourConvex(res[0])
+	if convex == true {
+		t.Errorf("Invalid IsContourConvex test")
```

Comment:
Why is this invalid?

Reply:
Just following convention of other tests - basically, the test is a failure because the contour in question is expected not to be convex.